src/openrct2/world/Sprite.h

```diff
@@ -13,14 +13,10 @@
 #include "../common.h"
 #include "../peep/Peep.h"
 #include "../ride/Vehicle.h"
+#include "EntityList.h"
```

Comment:
There is no need to include this here. Include it where it's used.

Reply:
it was used for `GetEntity`, but I handled that now